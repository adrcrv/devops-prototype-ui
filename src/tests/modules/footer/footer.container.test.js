import React from 'react';
import { render } from '@testing-library/react';
import FooterContainer from '../../../app/modules/footer/footer.container';

describe('Home', () => {
  it('Renders homepage unchanged', () => {
    const { container } = render(<FooterContainer />);
    expect(container).toMatchSnapshot();
  });
});
