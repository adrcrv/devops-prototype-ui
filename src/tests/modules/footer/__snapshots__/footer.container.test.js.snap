// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Renders homepage unchanged 1`] = `
<div>
  <div
    class="footer"
  >
    <div
      class="copyright"
    >
      Â© DevOps TCC. 2022. Todos os direitos reservados.
    </div>
  </div>
</div>
`;
